cli/command/trust/inspect_pretty_test.go

```diff
@@ -440,3 +441,20 @@ func TestFormatAdminRole(t *testing.T) {
 	targetsRoleWithSigs := client.RoleWithSignatures{Role: targetsRole, Signatures: nil}
 	assert.Check(t, is.Equal("Repository Key:\tabc, key11, key99\n", formatAdminRole(targetsRoleWithSigs)))
 }
+
+func TestPrintSignerInfoSortOrder(t *testing.T) {
+	roleToKeyIDs := map[string][]string{
+		"signer2-foo":  {"B"},
+		"signer10-foo": {"C"},
```

Comment:
nit; might want to swap `A`, `B` and `C` in this list, so that it's more clear that the list is sorted by `signer`, not `keys`